ntbs-service/Middleware/AuditGetRequestMiddleware.cs

```diff
@@ -0,0 +1,33 @@
+using System;

```

Comment:
So Postsharp did not work after all as it's not free but requires a license :(
I looked at other AOP frameworks, but as in our case we specifically want to intercept GET requests, I decided that it made more sense to just use normal middleware and add this to the pipeline in startup. It will run before handling it with the normal MVC framework.